config/locales/assessor_interface.en.yml

```diff
@@ -191,6 +191,7 @@ en:
           registration_number: We could not find the applicant’s reference number, or the number was in the wrong format. The applicant will need to supply the number again.
           registration_number_alternative: We could not find the applicant’s reference number. The applicant will need to upload written proof of recognition as a teacher instead.
           satisfactory_evidence_work_history: The information provided on work history is not sufficient to award QTS.
+          cannot_verify_details_of_a_school: We cannot verify details of a school that the applicant has entered
```

Comment:
We're adding this failure reason but I think it's missing from the `work_history_section` of the `AssessmentFactory`.

Reply:
Isnt that covered under SCHOOL_DETAILS_CANNOT_BE_VERIFIED